src/Parser/BloodDeathKnight/Modules/Features/BoneShieldUptime.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+
+import Module from 'Parser/Core/Module';
+import SPELLS from 'common/SPELLS';
+import Icon from 'common/Icon';
+
+import { formatPercentage } from 'common/format';
+import { formatDuration } from 'common/format';
+import StatisticBox, { STATISTIC_ORDER } from 'Main/StatisticBox';
+
+class BoneShieldUptime extends Module {
+
+
+  statistic() {
+
+    const boneshieldUptime = this.owner.modules.combatants.getBuffUptime(SPELLS.BONE_SHIELD.id);
+    const boneshieldUptimePercentage = boneshieldUptime / this.owner.fightDuration;
+
+    return (
+      <StatisticBox
+        icon={<Icon icon="ability_deathknight_boneshield" alt="Bone Shield" />}
+        value={`${formatPercentage(boneshieldUptimePercentage)}%`}
+        label='Bone Shield Uptime'
+        tooltip={`Bone Shield total uptime was ${formatDuration(boneshieldUptime / 1000)} seconds.`}
+      />
+
+    );
+  }
+  statisticOrder = STATISTIC_ORDER.CORE(1);
```

Comment:
This and Ossuary probably shouldn't have the same order value passed in.

Reply:
I had put them all to one because i was testing and trying to figure out what that order meant. Can you explain it to me? And if i wanted to order those stat boxes in a certain order how do i do that?

Reply:
Fixed